pkg/resources/view_acceptance_test.go

```diff
@@ -132,10 +107,9 @@ func TestAcc_View_basic(t *testing.T) {
 			},
 			// import - without optionals
 			{
-				ConfigDirectory: acc.ConfigurationDirectory("TestAcc_View/basic"),
-				ConfigVariables: basicView(statement),
-				ResourceName:    "snowflake_view.test",
-				ImportState:     true,
+				Config:       accconfig.ResourceFromModel(t, basicViewModel),
+				ResourceName: "snowflake_view.test",
```

Comment:
Nit: use model.ResourceReference(), in other places as well.

Reply:
Done.